.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,6 @@
+<!--
+!!!!!!! MAIN IS THE ONLY ACTIVE BRANCH. MAKE SURE THIS PR IS TARGETING MAIN. !!!!!!! 
```

Comment:
I realize there are a small set of PRs currently targeting the GA branch but the creator of those PRs explicitly know they should be targeting GA.